src/scripts/main.js

```diff
@@ -1,3 +1,43 @@
 'use strict';
 
-// write code here
+const th = document.querySelectorAll('th');
+const tbody = document.querySelector('tbody');
+const tr = [...tbody.querySelectorAll('tr')];
+
+th.forEach((header) => {
+  header.addEventListener('click', () => {
+    if (header.textContent === 'Name') {
+      tr.sort((firstElement, secondElement) =>
+        firstElement.cells[0].textContent
+          .toLowerCase()
+          // eslint-disable-next-line prettier/prettier
+          .localeCompare(secondElement.cells[0].textContent.toLowerCase()));
+    }
+
+    if (header.textContent === 'Position') {
+      tr.sort((firstElement, secondElement) =>
+        firstElement.cells[1].textContent
+          .toLowerCase()
+          // eslint-disable-next-line prettier/prettier
+          .localeCompare(secondElement.cells[1].textContent.toLowerCase()));
```

Comment:
Similar to the 'Name' column, consider implementing a toggle feature for the 'Position' column to allow sorting in both ascending and descending order.